### ISSUE TITLE
Push Notifications display image and name

### ISSUE THREAD

**Issue by garrethdev** [GrooveOtter/grooveotter-client#179]
Currently the system shows "Someone has liked your task" it should state that the person's full name and their image in the notification. 
![3 2](https://cloud.githubusercontent.com/assets/5047981/8779873/9e4e2b18-2ed3-11e5-803e-6e0e5b01bcb3.png)

**Comment by Ryan-Parker** [comment]
This is a repeat of #133

**Comment by garrethdev** [comment]
#133 has been closed

**Comment by garrethdev** [comment]
@L8D  Current issue is that when a person likes your task it shows your face instead of the person who liked it.

**Comment by L8D** [comment]
The change has been rolled back from the production server but can be tested here: https://gotr-staging.herokuapp.com/

**Comment by Ryan-Parker** [comment]
This has been tested cross browser and works. @L8D Please open a pull request and I will merge it into the master branch. Once that is done I will close and verify this ticket.

**Comment by L8D** [comment]
@Ryan-Parker things got a little mixed up earlier today which was why the changes were on staging (instead of a pull request server). Technically the changes merged into master when you merged #192
